components/cohort/Edit.vue

```diff
@@ -86,36 +96,52 @@ import 'carbon-web-components/es/components/button/index';
 import 'carbon-web-components/es/components/checkbox/index';
 import 'carbon-web-components/es/components/file-uploader/index';
 import IconUpload from '@carbon/icons-vue/es/upload/16';
-import type { Cohort } from '@/types/cohort';
+import IconSend from '@carbon/icons-vue/es/send/16';
+import type { Cohort, Member } from '@/types/cohort';
 
 const props = defineProps<{
   cohort: Cohort;
 }>();
 
 const cohortName = ref(props.cohort.name);
 const cohortMembers = ref([...props.cohort.members]);
+const cohortInvitations = ref([...props.cohort.invitations]);
 const addMemberInputText = ref('');
 const isAutoEnrollmentEnabled = ref(props.cohort.autoEnrollment);
 
-const informationMessage = reactive({
-  text: '',
-  isError: false,
-});
+const deletedMembersList = [] as Member[];
 
 const emits = defineEmits<{
+  (e: 'changed', data: { cohort: Cohort; deletedMembersList: Member[] }): void;
+  (e: 'cancelRequest'): void;
   (
-    e: 'changed',
-    data: {
-      cohort: Cohort;
-    }
+    e: 'sendRequest',
+    data: { cohort: Cohort; deletedMembersList: Member[] }
   ): void;
 }>();
 
+const informationMessage = reactive({
```

Comment:
I think this is used for inline validation messages? could we rename to

```suggestion
const inlineValidationMessage = reactive({
```